### PR TITLE
Add simple-git-hooks plugin

### PR DESCRIPTION
**Pull request by hyoban**

### CODE REVIEW

File: packages/knip/test/plugins/simple-git-hooks.test.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    ...baseArguments,
+    cwd,
+  });
+
+  // console.log(issues);
+
+  assert.deepEqual(counters, {
+    ...baseCounters,
+    binaries: 2,
+    devDependencies: 2,
```
Comment (webpro) [inline packages/knip/test/plugins/simple-git-hooks.test.ts:21]: Any chance you could add assertions about what binaries and devDependencies are considered issues here? E.g. `assert(issues.binaries['file']['executable']`?

**Review by webpro — APPROVED** [review]
Nice, thank you! Great PR, just one nitpick.

File: packages/knip/test/plugins/simple-git-hooks.test.ts (hunk 1 above)
Comment (hyoban) [reply packages/knip/test/plugins/simple-git-hooks.test.ts:24]: Updated, thanks for your review.

**Comment by webpro** [comment]
Thanks @hyoban!

**Comment by webpro** [comment]
:rocket: _This pull request is included in v5.20.0. See [Release 5.20.0](https://github.com/webpro-nl/knip/releases/tag/5.20.0) for release notes._

_Using Knip in a commercial project? Please consider [becoming a sponsor](https://knip.dev/sponsors)._

**Pull request merged**